src/Utilities/Utilities.jl

```diff
@@ -552,6 +553,12 @@ newlines(other) = 0
 
 # Output redirection.
 # -------------------
+@static if VERSION < v"0.7.0-DEV.3951"
+    link_pipe!(pipe; reader_supports_async = true, writer_supports_async = true) =
+        Base.link_pipe(pipe, julia_only_read = reader_supports_async, julia_only_write = writer_supports_async)
+else
+    import Base: link_pipe!
+end
```

Comment:
I don't think this would be very useful in Compat, so made this local fix here instead.